### PR TITLE
docs: tell Yarn users to match baseUrl of registry

### PR DESCRIPTION
**Pull request by HonkingGoose**
## Changes:

- Improve Azure devops docs with Yarn instructions

## Context:

@viceice asked me to put their working example in the Azure devops docs in https://github.com/renovatebot/renovate/discussions/9310#discussioncomment-546589.

I've put the config example plus some text in a new section for Yarn users, but maybe this is meant to be a general instruction?

## Documentation (please check one with an [x])

- [x] I have updated the documentation, or
- [ ] No documentation update is required

## How I've tested my work (please tick one)

I have verified these changes via:

- [x] Code inspection only, or
- [ ] Newly added unit tests, or
- [ ] No new tests but ran on a real repository, or
- [ ] Both unit tests + ran on a real repository

### CODE REVIEW

File: docs/usage/setup-azure-devops.md [hunk 1]
```diff
@@ -79,6 +79,32 @@ module.exports = {
...
+    },
+    {
+      domainName: 'github.com',
+      token: process.env.GITHUB_COM_TOKEN,
+    },
+  ],
+  repositories: ['YOUR-PROJECT/YOUR-REPO'],
+};
+```
+
```
Comment (viceice) [inline docs/usage/setup-azure-devops.md:107]:
```suggestion
repo **.npmrc**
    ```ini
    registry=https://<redacted>.pkgs.visualstudio.com/_packaging/<redacted>/npm/registry/
    always-auth=true
    ```
```

Don't know how to sucessfully escape the suggestion, but we should also add the required `.npmrc` settings

Comment (HonkingGoose) [reply docs/usage/setup-azure-devops.md:107]: I've copy/pasted your example into the docs. 😉

File: docs/usage/setup-azure-devops.md [hunk 2]
```diff
@@ -79,6 +79,39 @@ module.exports = {
...
 
+### Yarn users
+
+To do a succesfull `yarn install` you need to match the URL of the registry fully.
+Use the `baseUrl` config option to specify the full path to the registry.
+
+```javascript
+module.exports = {
+  platform: 'azure',
+  endpoint: 'https://<redacted>.visualstudio.com/',
```
Comment (HonkingGoose) [inline docs/usage/setup-azure-devops.md:90]: Do we want to keep using `<redacted>` in the Yarn configuration examples? The examples in the rest of the docs do not use `<redacted>`?

Would it be better to be consistent throughout maybe? Or is there a reason we use `<redacted>` in the Yarn configuration examples?

Comment (viceice) [reply docs/usage/setup-azure-devops.md:90]: maybe replace with something like `myorg` ?

**Review by viceice — APPROVED** [review]

**Pull request merged**